services/execsrv/store.go

```diff
@@ -157,15 +169,29 @@ func (p postgresStore) List(projectID *string, filterProvider *types.Provider, f
 			return []types.Exec{}, err
 		}
 
-		res[idx] = dbExecToExec(exec, dbSSHKeyToSSHKey(keys))
+		volumes, err := db.Q.ExecVolumeGet(ctx, exec.ID)
+		if err != nil {
+			return nil, err
+		}
+
+		res[idx] = dbExecToExec(exec, dbExecVolumesToVolumes(volumes), dbSSHKeyToSSHKey(keys))
 	}
 
 	return res, nil
 }
 
-func (p postgresStore) Delete(project, id string) error {
-	//TODO implement me
-	panic("implement me")
+func (p postgresStore) Delete(_, id string) error {
```

Comment:
Let's delete project from interface definition if not used

Reply:
No worries just wasn't sure if we wanted to keep this around for any reason